lib/locator.js

```diff
@@ -472,24 +469,6 @@ function isShadow(locator) {
   return hasShadowProperty;
 }
 
-/**
```

Comment:
this wasnt needed because isCustom exists already to cover this use case in locators.js